lib/devise/multi_email/models/authenticatable.rb

```diff
@@ -1,21 +1,23 @@
 module Devise
   module Models
     module EmailAuthenticatable
-      USER_ASSOCIATION = :user
+      # deprecated
+      USER_ASSOCIATION = Devise::MultiEmail.parent_association_name
 
       def devise_scope
-        user_association = self.class.reflect_on_association(USER_ASSOCIATION)
+        user_association = self.class.reflect_on_association(Devise::MultiEmail.parent_association_name)
         if user_association
           user_association.class_name.constantize
         else
-          raise "#{self.class.name}: Association :#{USER_ASSOCIATION} not found: Have you declared that ?"
+          raise "#{self.class.name}: Association :#{Devise::MultiEmail.parent_association_name} not found: Have you declared that ?"
         end
       end
     end
 
     module MultiEmailAuthenticatable
       extend ActiveSupport::Concern
-      EMAILS_ASSOCIATION = :emails
+      # deprecated
+      EMAILS_ASSOCIATION = Devise::MultiEmail.emails_association_name
```

Comment:
😄  Can you help remove this constant as well ?  I suspect it might cause problems if the constant gets defined before the configuration was set.